demisto_sdk/commands/common/hook_validations/mapper.py

```diff
@@ -47,6 +47,51 @@ def is_valid_version(self):
         """
         return self._is_valid_version()
 
+    def is_backward_compatible(self):
+        # type: () -> bool
+        """Check whether the Mapper is backward compatible or not, update the _is_valid field to determine that"""
+
+        answers = [
+            self.is_changed_incidents_fields(),
```

Comment:
`changed` is not a proper word as we allow changes (in the form of addition).
better to call it `is_field_mapping_removed`